command/cp.go

```diff
@@ -438,12 +469,11 @@ func (c Copy) Run(ctx context.Context) error {
 		default:
 			panic("unexpected src-dst pair")
 		}
-
 		parallel.Run(task, waiter)
 	}
-
 	waiter.Wait()
 	<-errDoneCh
+	c.progressbar.Finish()
```

Comment:
It'd be more readable if you call any cleanup or finish-like functions with `defer` right after the initialization call.

Reply:
Resolved this and the others. https://github.com/peak/s5cmd/pull/590/commits/ad85af2f9aeef0f187f1f98f373d9da415770c75

Reply:
Thank you 🚀